pandas/plotting/_core.py

```diff
@@ -468,12 +468,13 @@ def _adorn_subplots(self):
                 self.axes[0].set_title(self.title)
 
     def _apply_axis_properties(self, axis, rot=None, fontsize=None):
-        labels = axis.get_majorticklabels() + axis.get_minorticklabels()
-        for label in labels:
-            if rot is not None:
-                label.set_rotation(rot)
-            if fontsize is not None:
-                label.set_fontsize(fontsize)
+        if rot is not None or fontsize is not None:
```

Comment:
can you add a comment here (doc-string is a + as well)

Reply:
Done